### PR TITLE
docs: added 8.21 Shanghai meetup blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

docs: added 8.21 Shanghai meetup blog

### CODE REVIEW

File: website/blog/2021-08-21-shanghai-meetup.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### 基于 Apache APISIX，爱奇艺 API 网关的落地实践
+
+#### 分享嘉宾
+
+蒋文杰 / 爱奇艺科学家
+
+#### 议题详情
+
+API 网关已经成为微服务架构中不可或缺的一环。它既承担着服务对外的唯一门户，又需要满足许多应用的共性功能。爱奇艺作为一家在线视频公司，每天需要承载千万级的调用量，并且对数据的安全、用户请求响应时效、系统的稳定都有着极其严格的要求，因此公司需要打造一个高性能、高可用的 API 网关。本文简单介绍爱奇艺 API 网关的架构以及实现的功能，包括请求分发、条件路由、Api 管理、限流熔断、安全三防、监控报警以及全链路追踪等。
```
Comment (liuxiran) [inline website/blog/2021-08-21-shanghai-meetup.md]:
```suggestion
API 网关已经成为微服务架构中不可或缺的一环。它既承担着服务对外的唯一门户，又需要满足许多应用的共性功能。爱奇艺作为一家在线视频公司，每天需要承载千万级的调用量，并且对数据的安全、用户请求响应时效、系统的稳定都有着极其严格的要求，因此公司需要打造一个高性能、高可用的 API 网关。本文简单介绍爱奇艺 API 网关的架构以及实现的功能，包括请求分发、条件路由、API 管理、限流熔断、安全三防、监控报警以及全链路追踪等。
```

File: website/blog/2021-08-21-shanghai-meetup.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: "8 月 21 日 Apache APISIX Meetup 上海站，欢迎报名参加！"
+---
+
+<!--truncate-->
+本次 Meetup 由 [API7.AI](https://www.apiseven.com/) 发起，并邀请爱奇艺、空中云汇等合作伙伴共同为大家呈现。Apache APISIX PMC 成员、贡献者、社区技术专家汇聚一堂，共同探讨 Apache APISIX 社区发展、行业实践等话题。
```
Comment (juzhiyuan) [inline website/blog/2021-08-21-shanghai-meetup.md]:
```suggestion
本次 Meetup 由 [api7.ai](https://www.apiseven.com/) 发起，并邀请爱奇艺、空中云汇等合作伙伴共同为大家呈现。Apache APISIX PMC 成员、贡献者、社区技术专家汇聚一堂，共同探讨 Apache APISIX 社区发展、行业实践等话题。
```

File: website/blog/2021-08-21-shanghai-meetup.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+#### 议题详情
+
+API 网关已经成为微服务架构中不可或缺的一环。它既承担着服务对外的唯一门户，又需要满足许多应用的共性功能。爱奇艺作为一家在线视频公司，每天需要承载千万级的调用量，并且对数据的安全、用户请求响应时效、系统的稳定都有着极其严格的要求，因此公司需要打造一个高性能、高可用的 API 网关。本文简单介绍爱奇艺 API 网关的架构以及实现的功能，包括请求分发、条件路由、Api 管理、限流熔断、安全三防、监控报警以及全链路追踪等。
+
+### 如何使用 Apache APISIX 进行集中式身份认证
+
+#### 分享嘉宾
+
+朱欣欣 / API7 工程师，Apache APISIX Committer
```
Comment (juzhiyuan) [inline website/blog/2021-08-21-shanghai-meetup.md]:
```suggestion
朱欣欣 / api7.ai 工程师，Apache APISIX Committer
```

File: website/blog/2021-08-21-shanghai-meetup.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+#### 议题详情
+
+数据无处不在，每时每刻都有海量数据在世界各地被处理、传输、存储。然而数据世界不是法外之地，数据的收集与处理必须遵照各国法律执行。这为企业的技术架构提出了巨大的挑战，如何兼顾用户体验、整洁架构与数据主权？API 网关在企业架构中具有特殊地位，也因此能在数据主权场景下起到特殊重要的作用。
+
+### 独行者速，众行者远，Apache APISIX 感恩有你
+
+#### 分享嘉宾
+
+王院生 / API7.AI 联合创始人 & CTO
```
Comment (juzhiyuan) [inline website/blog/2021-08-21-shanghai-meetup.md]:
```suggestion
王院生 / api7.ai 联合创始人 & CTO
```

**Review by juzhiyuan — APPROVED** [review]
Why not put this in Event?

**Review by liuxiran — APPROVED** [review]

**Comment by liuxiran** [comment]
> Why not put this in Event?

good idea, I noticed all events in our events list are about releases, our meetup ad is also a big event for our community.

**Review by bzp2010 — APPROVED** [review]

**Comment by juzhiyuan** [comment]
image

**Review by tokers — APPROVED** [review]

**Comment by Baoyuantop** [comment]
https://github.com/apache/apisix-ingress-controller/pull/628

@juzhiyuan @liuxiran

**Pull request merged**